pkgs/development/python-modules/pyunpack/default.nix

```diff
@@ -1,16 +1,18 @@
 { lib
 , buildPythonPackage
 , fetchFromGitHub
+, setuptools
 , pytestCheckHook
 , easyprocess
 , entrypoint2
 , patool
+, cabextract
 }:
 
 buildPythonPackage rec {
   pname = "pyunpack";
   version = "0.3";
-  format = "setuptools";
+  pyformat = true;
```

Comment:
what is `pyformat`.  it doesn't seem to exist in nixpkgs.

Reply:
Thanks, my brain seems to have short-circuited. :)